app/src/main/res/layout/activity_main2.xml

```diff
@@ -19,13 +20,11 @@
         app:layout_constraintBottom_toTopOf="@+id/imageView"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        android:autofillHints="" />
```

Comment:
If you use "android:autofillHints" you need to specify what type you use for autocomplete, 
for example "android:autofillHints=text"